lib/teleterm/clusters/storage.go

```diff
@@ -99,7 +99,7 @@ func (s *Storage) Add(ctx context.Context, webProxyAddress string) (*Cluster, er
 	clusterName := parseName(webProxyAddress)
 	for _, pname := range profiles {
 		if pname == clusterName {
-			return nil, trace.BadParameter("cluster %v already exists", clusterName)
+			return s.GetByName(pname)
```

Comment:
Let's use `Storage.fromProfile` here and remove `GetByName` completely. It's not used anywhere and also we know now that it's incorrect, [the proxy address does not equal the name of the cluster](https://github.com/gravitational/webapps.e/issues/261).

`Storage.GetByName` could be renamed to `Storage.GetByProfileName`, but since it's not used anywhere I think we should just remove it. If it becomes useful we can easily reintroduce it.